scripts/deploy-kubefed.sh

```diff
@@ -153,19 +155,14 @@ check-command-installed helm
 # Build KubeFed binaries and image
 if [[ "${USE_LATEST:-}" != "y" ]]; then
   cd "$(dirname "$0")/.."
-  make container IMAGE_NAME=${IMAGE_NAME}
+  make container IMAGE_NAME="${IMAGE_NAME}"
   cd -
-  # Use DOCKER_PUSH=n ./scripts/deploy-kubefed.sh <image> to skip docker
-  # push on container image when not using latest image.
-  if [[ "${DOCKER_PUSH:-y}" == "y" ]]; then
```

Comment:
This variable is still passed in from the [Makefile](https://github.com/kubernetes-sigs/kubefed/blob/59e60d97b5b79ff3b2dfe73bbecaa188938be372/Makefile#L169)

Reply:
Removed, good catch!